api/accounts_api.py

```diff
@@ -13,17 +13,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-
-
 import logging
 
 from google.cloud import ndb
 
 from framework import basehandlers
 from framework import permissions
 from framework import ramcache
-from internals import models
+from internals import user_models
+
+
+def user_to_json_dict(user):
+  return {
+      'is_admin': user.is_admin,
+      'is_site_editor': user.is_site_editor,
+      'email': user.email,
+      'id': user.key.integer_id(),
+  }
+
```

Comment:
Out of curiosity, what's the motivation to not having this as a method in the `AppUser` class?

Reply:
I feel that the API code should have responsibility for protocol between the client and server, including the serialization format.  The model classes should just have the schema, database-level validation, and basic queries.